bucket/DIE.json

```diff
@@ -0,0 +1,47 @@
+{
+    "description": "Detect It Easy, or abbreviated \"DIE\" is a program for determining types of files.",
+    "homepage": "https://ntinfo.biz/index.html",
+    "license": "MIT",
+    "version": "2.05",
+    "url": "https://github.com/horsicq/DIE-engine/releases/download/2.05/die_win32_portable_2.05.zip",
+    "hash": "c597381d73a7ce6ad1acc216c5ddd9010cde56c830063eaeee9411974f60963e",
+    "extract_dir": "die_win32_portable/base",
+    "installer": {
+        "script": [
+            "if (Test-Path \"$persist_dir\\die.ini\") { Copy-Item \"$persist_dir\\die.ini\" \"$dir\" -Force }",
+            "if (Test-Path \"$persist_dir\\diec.ini\") { Copy-Item \"$persist_dir\\diec.ini\" \"$dir\" -Force }"
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "New-Item \"$persist_dir\" -ItemType Directory -ErrorAction SilentlyContinue | Out-Null",
+            "if (Test-Path \"$dir\\die.ini\") { Copy-Item \"$dir\\die.ini\" \"$persist_dir\" -Force }",
+            "if (Test-Path \"$dir\\diec.ini\") { Copy-Item \"$dir\\diec.ini\" \"$persist_dir\" -Force }"
```

Comment:
```suggestion
            "ensure \"$persist_dir\" | Out-Null",
            "Copy-Item \"$dir\\*\" \"$persist_dir\" -Include 'die.ini', 'diec.ini' -ErrorAction SilenltyContinue -Force"
```

Reply:
Is `ensure` documented somewhere? Is that part of scoop? or is it provided by powershell?

I've seen other places defining their own `ensureFile` function, is that a legacy thing or does this `ensure` only support directories?

Reply:
https://github.com/lukesampson/scoop/blob/08af9ff6e7f7b017701c0c9114294b13e1e83fb8/lib/core.ps1#L354

Reply:
Cool, thanks